app/src/main/java/com/yacgroup/yacguide/AboutActivity.kt

```diff
@@ -36,12 +38,33 @@ class AboutActivity : BaseNavigationActivity() {
         super.onCreate(savedInstanceState)
         this.title = getString(R.string.menu_about, getString(R.string.app_name))
 
-        val textViewAppVersion: TextView = findViewById(R.id.textViewAppVersion)
-        textViewAppVersion.text = "Version ${getAppVersion()}"
-        textViewAppVersion.gravity = Gravity.CENTER_HORIZONTAL
+        _createEntry(getString(R.string.data_src), getString(R.string.data_src_url),null)
+        _createEntry(getString(R.string.privacy_policy), null) { _showPrivacyPolicy() }
```

Comment:
Shouldn't `{ _showPrivacyPolicy() }` be part of the method's argument list?

Reply:
Android Studio suggestions: _Lambda argument should be moved out of parentheses_ . It is a bit strange syntax. There is a another way to get rid of this warning.